src/core_modules/capture-core/components/FiltersForTypes/Date/DateFilter.component.js

```diff
@@ -191,32 +177,25 @@ class DateFilterPlain extends Component<Props, State> implements UpdatableFilter
     }
 
     static isFromAfterTo(valueFrom: string, valueTo: string) {
-        const momentFrom = parseDate(valueFrom).momentDate;
-        const momentTo = parseDate(valueTo).momentDate;
-        // $FlowFixMe[incompatible-use] automated comment
-        // $FlowFixMe[incompatible-call] automated comment
-        return momentFrom.isAfter(momentTo);
+        const dateFrom = new Date(valueFrom);
```

Comment:
The `The From date can't be after the To date` error is missing when the calendar is in the dd-MM-yyyy format. Below is a video with the steps on the branch vs play. 

https://github.com/user-attachments/assets/c914412c-e728-4c78-a563-1a3c67c827bb

Reply:
Probably ok to keep using moment for now? Why did we change this part?

Reply:
@JoakimSM Because I wanted to remove `parseDate` function from parsers folder at the end if it isn't used anywhere else.

Reply:
I put it back since it results in invalid dates for dd-mm-yyyy format